components/create_comment/create_comment.tsx

```diff
@@ -573,6 +574,7 @@ class CreateComment extends React.PureComponent<Props, State> {
         let channelTimezoneCount = 0;
         let mentions: string[] = [];
         const notContainsAtChannel = !containsAtChannel(draft.message);
+        const containsAtHereMention = notContainsAtChannel && containsAtChannel(draft.message, {checkAllMentions: true});
```

Comment:
Since we're treating at-here the same as at-channel now, could we just have `containsAtChannel` return true for at-here as well? Perhaps `containsAtChannel` could also return which of those are matched so that we don't need to work it out below.

Reply:
Yeah logic there is getting a bit complex. I'll have a look.